kpm/lib/kpm/nexus_helper/nexus_api_calls_v2.rb

```diff
@@ -27,7 +27,7 @@ def message
     class NexusApiCallsV2
       PULL_ARTIFACT_ENDPOINT = '/service/local/artifact/maven/redirect'
       GET_ARTIFACT_INFO_ENDPOINT = '/service/local/artifact/maven/resolve'
-      SEARCH_FOR_ARTIFACT_ENDPOINT = '/service/local/data_index'
+      SEARCH_FOR_ARTIFACT_ENDPOINT = '/service/local/lucene/search'
```

Comment:
The "/data_index" does not return the complete list of versions at the moment of search, for example the latest version for Killbill with the endpoint "/data_index" gave me 0.18.6 and with "/lucene/search" gave me 0.18.12. Also the mount point "/data_index" is deprecated.